Example/package.json

```diff
@@ -7,7 +7,7 @@
   },
   "dependencies": {
     "react": "~15.3.0",
-    "react-native": "^0.34.0",
+    "react-native": "^0.41.2",
```

Comment:
Have you tried out IOS in the example app since upgrading this? All headers have moved from global RCT* to React/.